### ISSUE TITLE
Map-level conkeyref-push of a topic does not work

### ISSUE THREAD

**Issue by chrispy-snps** [dita-ot/dita-ot#3711]
## Expected Behavior
According to the DITA spec at

https://docs.oasis-open.org/dita/dita/v1.3/os/part2-tech-content/archSpec/base/conref-overview.html

a conref/conkeyref can target:

- A referenced element within a DITA map
- A referenced element within a DITA topic
- **An entire DITA map**
- **An entire DITA topic**

Thus, I expect that this mechanism would work on an entire topic within a map:

```
<!-- attempt to replace topic 2 by topic key -->
<topicref href="topic_2_replacement.dita" processing-role="resource-only" conaction="pushreplace" conkeyref="topic_2"/>
<topicref href="topic_2.dita" keys="topic_2"/>
```

## Actual Behavior
The DITA-OT issues the following error:

`[conref-push] [DOTJ041E][ERROR] The attribute conref="topic_2.dita" uses invalid syntax. The value should contain '#' followed by a topic or map ID, optionally followed by '/elemID' for a sub-topic element.`

## Steps to Reproduce
For the following testcase:

[ditaot_topic_override.zip](https://github.com/dita-ot/dita-ot/files/6050153/ditaot_topic_override.zip)

run the following command:

`dita -i map.ditamap -f html5`

## Environment
* DITA-OT version: 3.6.0
* Operating system and version: linux
* How did you run DITA-OT? `dita` command
* Transformation type: `html5`

**Comment by raducoravu** [comment]
I think there is a misunderstanding here, when you use "conaction", "conref" or "conkeyref" on a topicref, you do that to reuse another "topicref" element or to replace another "topicref" element from the DITA Map context. So a "pushreplace" on a "topicref" means that you want to replace another specific "topicref", not that you want to replace the contents of a topic with the contents of another topic.
If I were to want to push the contents of a topic over the contents of another topic, I would need to specify the conkeyref push directly inside that topic file:

    <topic id="id1" conkeyref="topic_2" conaction="pushreplace">
       <title>Topic 1</title>
       <body>
           <p>This is topic 1.</p>
       </body>
    </topic>

and this also does not work. I'm attaching a small project with how I considered things should work:
[conref-push-topic.zip](https://github.com/dita-ot/dita-ot/files/6061138/conref-push-topic.zip)

**Comment by chrispy-snps** [comment]
Thanks @raducoravu !

Using your simpler testcase, I see that replacement of individual elements _within_ a topic works:

```
<topic id="id1">
    <title conkeyref="topic_2/id2_title" conaction="pushreplace">Topic 1</title>
    <body conkeyref="topic_2/id2_body" conaction="pushreplace">
        <p>This is topic 1.</p>
    </body>
</topic>
```

but this cannot be used to reliably replace one topic with another when the original topic has additional elements (`abstract`, `related-links`, etc.) that the replacement does not.

**Comment by raducoravu** [comment]
@chrispy-snps yes, it would be better if the conkeyref push from the topic root element worked.
